config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -759,26 +759,9 @@ def generate_misc():
                        "--instance-manager=karpenter",
                        "--master-size=c6g.xlarge",
                    ],
-                   feature_flags=['Karpenter'],
                    extra_dashboards=["kops-misc"],
                    skip_regex=r'\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|nfs|NFS|Gluster|Services.*rejected.*endpoints|TCP.CLOSE_WAIT|external.IP.is.not.assigned.to.a.node|same.port.number.but.different.protocols|same.hostPort.but.different.hostIP.and.protocol|should.create.a.Pod.with.SCTP.HostPort|Services.should.create.endpoints.for.unready.pods|Services.should.be.able.to.connect.to.terminating.and.unready.endpoints.if.PublishNotReadyAddresses.is.true|should.verify.that.all.nodes.have.volume.limits|In-tree.Volumes|LoadBalancers.should.be.able.to.preserve.UDP.traffic|serve.endpoints.on.same.port.and.different.protocols'), # pylint: disable=line-too-long
 
-        build_test(name_override="kops-aws-ipv6-karpenter",
```

Comment:
Why remove this test?

Reply:
This fails to even start.

Reply:
It's useful signal. We should create issues to fix Karpenter on IPv6 clusters.

Reply:
Added back